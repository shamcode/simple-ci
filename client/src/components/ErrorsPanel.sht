{% if errors.length > 0 %}
    <ul class="errors-panel">
        {% for error of errors %}
            <li>{{error}}</li>
        {% endfor %}
    </ul>
{% endif %}
