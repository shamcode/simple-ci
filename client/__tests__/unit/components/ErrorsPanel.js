import ErrorsPanel from '../../../src/components/ErrorsPanel.sht';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( ErrorsPanel, {} );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
