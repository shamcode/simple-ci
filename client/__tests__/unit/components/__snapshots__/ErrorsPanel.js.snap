// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Constructor": "ErrorsPanel",
  "Options": Object {
    "ID": "component",
  },
  "html": "<!--if-->",
}
`;
